test/utilities.js

```diff
@@ -17,6 +17,32 @@ let hexToBinary = function(value) {
     return Buffer.from(value, 'hex').toString('binary');
 };
 
+// helper function to send a UTXO on childchain and submit blockheader to rootchain.
+let sendUTXO = async function(rootchain, sender, blkNum1, txIndex1, oIndex1, amount1, blkNum2, txIndex2, oIndex2, amount2, newOwner1, denom1, newOwner2, denom2) {
```

Comment:
instead of passing in all the args. why not pass the rlp encoded txByes? The args aren't used after the encoding

Reply:
also removing the RLP require statement

Reply:
Ahh yeah, my b.